pod-batch.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+# this value should be differernt when running script again, as each time a script is run a new Pod is created
+  name: pod1
+spec:
+  containers:
+  - name: barepod
+  # Docker image name
+    image: 172.30.223.233:5000/sample-project/batch-cgs
+    command: ["python", "groupsforsections.py"]
+    env:
+    # the below 2 are environmental variables in Open Shift
```

Comment:
Having default values in not appropriate, so the script would refuse to run until good values provided